.circleci/workflows.yml

```
@@ -1527,7 +1529,7 @@ jobs:
     steps:
       - restore_cached_workspace
       - run:
-          command: yarn workspace @tooling/system-tests test:ci "test/non_root*spec*" --browser electron
+          command: CYPRESS_COMMERCIAL_RECOMMENDATIONS=0 yarn workspace @tooling/system-tests test:ci "test/non_root*spec*" --browser electron
```

```suggestion
          command:  yarn workspace @tooling/system-tests test:ci "test/non_root*spec*" --browser electron
```

I need to fix these in develop for them to apply